xmake/modules/detect/sdks/find_ifortenv.lua

```diff
@@ -100,8 +100,14 @@ function _find_intel_on_windows(opt)
     opt = opt or {}
 
     -- find ifortvars_bat.bat
-    local paths = {"$(env IFORT_COMPILER20)"}
+    local paths = {os.getenv("IFORT_COMPILER20")}
```

Comment:
Why use os.getenv instead of $(env)? This will cause environment variables to be calculated in advance. If the searched list is very large, they will be obtained in advance, if one of them returns nil, the table will be interrupted

Reply:
My bad, I thought this would be executed as a system command and there is no `env` command on windows. But this does indeed work.

Reply:
$(env) will also call os.getenv